tests/commands/test_commands.py

```diff
@@ -341,15 +343,74 @@ def json(self):
         return {'result': {'latest_block_height': self.height}}
 
 
-@pytest.mark.skip
-@patch('bigchaindb.config_utils.autoconfigure')
-@patch('bigchaindb.backend.query.store_validator_update')
-@pytest.mark.tendermint
-def test_upsert_validator(mock_autoconfigure, mock_store_validator_update):
-    from bigchaindb.commands.bigchaindb import run_upsert_validator
+# @pytest.mark.execute
+# @patch('bigchaindb.lib.BigchainDB.get_validators')
+# @pytest.mark.abci
+def test_upsert_validator_new_with_tendermint(b, priv_validator_path, user_sk, monkeypatch):
+    """
+    WIP: Will be fixed and activated in the next PR
+    """
+    from bigchaindb.commands.bigchaindb import run_upsert_validator_new
+    import time
+
+    time.sleep(3)
+
+    def mock_get():
+        return [
+            {'pub_key': {'value': 'zL/DasvKulXZzhSNFwx4cLRXKkSM9GPK7Y0nZ4FEylM=',
+                         'type': 'tendermint/PubKeyEd25519'},
+             'voting_power': 10}
+        ]
+
+    # b.get_validators = mock_get
+    # mock_get_validators = mock_get
+    # monkeypatch.setattr('requests.get', mock_get)
 
-    args = Namespace(public_key='CJxdItf4lz2PwEf4SmYNAu/c/VpmX39JEgC5YpH7fxg=',
-                     power='10', config={})
-    run_upsert_validator(args)
+    proposer_key = b.get_validators()[0]['pub_key']['value']
 
-    assert mock_store_validator_update.called
+    args = Namespace(action='new',
+                     public_key=proposer_key,
+                     power=1,
+                     node_id='12345',
+                     sk=priv_validator_path,
+                     config={})
+    print("*****ARGS*****")
```

Comment:
left over?

Reply:
Please remove these print statements.

Reply:
Those are in a WIP test that is still in development, for diagnostic purposes. It's the next item on the agenda for this branch, so please just ignore them for now.